Dockerfile.template

```diff
@@ -1,12 +1,12 @@
 ARG ARCH=%%BALENA_ARCH%%
 ARG FATRW_VERSION=0.2.9
-ARG NODE="nodejs<18"
-ARG NPM="npm<9"
+ARG NODE="nodejs>16"
```

Comment:
Is there not a better way to do this since this would to my understanding install nodejs 20?

Reply:
We can use ` nodejs<20`,  but the node version won't change unless we bump the alpine version.